### PR TITLE
fix: drop wrong position in tree

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
테스트

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jung-han — APPROVED** [review]
완료합니다!

File: packages/toast-ui.grid/src/view/bodyArea.tsx [hunk 1]
```diff
@@ -162,23 +162,29 @@ class BodyAreaComp extends Component<Props> {
...
+        this.movedIndexInfo = { index, rowKey: rowKeyToMove };
+        this.props.dispatch('moveRow', rowKey, index);
+      }
+
+      const gridEvent = new GridEvent({
+        rowKey,
+        targetRowKey: rowKeyToMove,
+        appended: !!this.movedIndexInfo!.appended,
+      });
+
```
Comment (shiren) [inline packages/toast-ui.grid/src/view/bodyArea.tsx:178]: 커밋이 잘못되었나? 이 수정은 두 개의 PR에 있네요?

**Review by shiren — APPROVED** [review]
수고하셨습니다.

File: packages/toast-ui.grid/src/view/bodyArea.tsx (hunk 1 above)
Comment (lja1018) [reply packages/toast-ui.grid/src/view/bodyArea.tsx:178]: 이전 PR에서 이어서 작업하셔서 그런거 같습니다.

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Review by jungeun-cho — APPROVED** [review]
이견없습니다.

File: packages/toast-ui.grid/src/view/bodyArea.tsx (hunk 1 above)
Comment (js87zz) [reply packages/toast-ui.grid/src/view/bodyArea.tsx:178]: 머지 대상 브랜치를 제가 잘못 땄네요..

**Pull request closed without merging**